charts/kubescape-operator/values.yaml

```diff
@@ -4,10 +4,23 @@
 
 ksNamespace: kubescape
 ksLabel: kubescape
-createKubescapeServiceAccount: true # TODO: move to kubescape
 
 capabilities:
-  relevancy: detect # enable/disable/detect
+  relevancy: enable
+  configurationScan: enable
+  continuousScan: disable
+  vulnerabilityScan: enable
+  nodeScan: enable
+  networkGenerator: disable
+  seccomp: disable
+  runtimeObservability: disable
+  otel: enable
```

Comment:
OTEL should be disabled by default.
If there is a "third-party" we should enable it by default.
```suggestion
  otel: disable
```

Reply:
I think I will move the otel-url to `configurations` and use that for enabling the capability and its configuration